packages/dds/tree/src/test/forestProxy.spec.ts

```diff
@@ -0,0 +1,73 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { strict as assert } from "assert";
```

Comment:
I recently added a feature-libraries folder inside `test` since we were getting a lot of top level test files. This should get moved in there.